spark/v3.4/spark/src/main/java/org/apache/iceberg/spark/SparkFunctionCatalog.java

```diff
@@ -16,23 +16,30 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+package org.apache.iceberg.spark;
 
```

Comment:
I wonder a little about the naming here, I know i'm always confused about SparkCatalog (our class) versus the Spark Catalog class. IcebergFunctionCatalog? I don't feel strongly about this though.

Reply:
I also thought a bit about it but then looked at all our other classes like `SparkCatalog`, `SparkWrite`, etc. I guess it makes sense to follow this patter because catalogs are referred using the qualified name that includes the package too.

Reply:
Yeah, it was a questionable decision on our end to go with this naming in the first place.